Lib/idlelib/idle_test/test_configdialog.py

```diff
@@ -141,13 +143,13 @@ def test_select_font_mouse(self):
         fontlist = dialog.fontlist
         fontlist.activate(0)
 
-      # Select next item in listbox
+        # Select next item in listbox
         fontlist.focus_force()
         fontlist.see(1)
         fontlist.update()
         x, y, dx, dy = fontlist.bbox(1)
-        fontlist.event_generate('<Button-1>', x=x+dx//2, y=y+dy//2)
-        fontlist.event_generate('<ButtonRelease-1>', x=x+dx//2, y=y+dy//2)
+        fontlist.event_generate('<Button-1>', x=x + dx // 2, y=y + dy // 2)
+        fontlist.event_generate('<ButtonRelease-1>', x=x + dx // 2, y=y + dy // 2)
```

Comment:
I disagree.  For an assignment statement, I would write
x = x + dx//2
Following math tradition here is allowed by pep8.  When the space around '=' is deleted, as per PEP8, it looks weird, and visually deceptive, to leave space around '+'.   I don't know if this is specifically addressed.